wazo_confd/sync_db.py

```diff
@@ -87,6 +92,13 @@ def main():
             tenant = tenant_dao.find_or_create_tenant(tenant_uuid)
             default_sip_template_service.generate_sip_templates(tenant)
 
+        for tenant_uuid in confd_tenant_without_slugs:
+            slug = auth_tenant_slugs.get(tenant_uuid)
+            if not slug:
+                continue
+            tenant = tenant_dao.find_or_create_tenant(tenant_uuid)
+            tenant.slug = slug
```

Comment:
by good practice we should flush session after this operation (with flush directly or wrap it in `tenant_dao.update(tenant)`)

Reply:
which does not exist :sweat_smile:

Reply:
Hahaha